pkg/api/vlabs/validate_test.go

```diff
@@ -2605,6 +2605,29 @@ func TestAgentPoolProfile_ValidateVirtualMachineScaleSet(t *testing.T) {
 	})
 }
 
+func TestAgentPoolProfile_ValidateAuditDEnabled(t *testing.T) {
```

Comment:
you're missing a UT for master profile validation

https://codecov.io/gh/Azure/aks-engine/compare/77822c37674a7ec81803a2dae6d2d082a20dd5ab...75a21cb62c4ad6e9af57c9d8cd8d1151da84b7dc/src/pkg/api/vlabs/validate.go

Reply:
done